mods/ts/audio/speech-generic.yaml

```diff
@@ -26,6 +26,7 @@ Speech:
 		CriticalStructureLost: 00-i196
 		CriticalUnitLost: 00-i194
 		DisablePower: 00-i230
+        DropPodsAvailable: 00-i506
```

Comment:
While the engine supports spaces in yaml, all our official mods use tabs, so please make sure to fix all places that use spaces.
If you have trouble identifying them locally, you can see it easily here on gh (red = tabs, no red = spaces).